### PR TITLE
Address sprint 34 feedback

### PR DESCRIPTION
**Pull request by matthopson**
Address feedback for Sprint 34: https://github.com/ustaxcourt/ef-cms/pull/155

### CODE REVIEW

File: shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.test.js [hunk 1]
```diff
@@ -43,17 +43,24 @@ describe('generateStandingPretrialNoticeTemplate', () => {
       },
     });
 
-    // TODO: Flesh this out further when template is done
     expect(result.indexOf('<!DOCTYPE html>')).toBe(0);
     expect(result.indexOf('Test Case Caption')).toBeGreaterThan(-1);
     expect(result.indexOf('123-45S')).toBeGreaterThan(-1);
+    expect(result.indexOf('Courthouse Name')).toBeGreaterThan(-1);
+    expect(result.indexOf('Address 1')).toBeGreaterThan(-1);
+    expect(result.indexOf('Address 2')).toBeGreaterThan(-1);
```
Comment (kkoskelin) [inline shared/src/business/useCaseHelper/standingPretrialNotice/generateStandingPretrialNoticeTemplate.test.js:51]: nit: rainy-day project to replace `.indexOf('substring')).toBeGreaterThan(-1)` with `.includes('substring')).toBeTruthy()`

File: web-client/src/presenter/actions/WorkItem/assignPetitionToAuthenticatedUserAction.test.js [hunk 2]
```diff
@@ -42,14 +42,14 @@ describe('assignPetitionToAuthenticatedUserAction', () => {
         caseDetail: {
           documents: [
             {
-              docuementType: INITIAL_DOCUMENT_TYPES.petition.documentType,
+              documentType: INITIAL_DOCUMENT_TYPES.petition.documentType,
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/WorkItem/assignPetitionToAuthenticatedUserAction.test.js:45]: 😮

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/actions/WorkItem/assignPetitionToAuthenticatedUserAction.test.js (hunk 2 above)
Comment (matthopson) [reply web-client/src/presenter/actions/WorkItem/assignPetitionToAuthenticatedUserAction.test.js:45]: 😬

**Comment by matthopson** [comment]
(yಠ,ಠ)y snyk.....

**Pull request merged**